navigator/backend/src/main/scala/com/digitalasset/navigator/config/Arguments.scala

```diff
@@ -69,29 +72,29 @@ object Arguments {
         Some(TlsConfiguration(true, Some(new File(path)), None, None)))(c =>
         Some(c.copy(keyCertChainFile = Some(new File(path))))))
 
-  private def argumentParser(defaultConfigFile: Path) =
+  private def argumentParser(defaultConfigFile: Path, defaultAccessTokenFile: Path) =
     new OptionParser[Arguments]("navigator") with ArgumentsHelper {
       help("help").abbr("h").text("prints this usage text")
 
       opt[Int]("port")
-        .text("port number on which the server should listen (default 4000)")
+        .text(s"port number on which the server should listen (default ${Arguments.default.port})")
         .action((port, arguments) => arguments.copy(port = port))
 
       opt[String]("assets")
         .text("folder where frontend assets are available")
         .action((assets, arguments) => arguments.copy(assets = Some(assets)))
 
-      opt[Unit]('w', "require-password")
-        .text("if true then the password is required to login")
-        .action((_, arguments) => arguments.copy(requirePassword = true))
+      opt[String]('t', "access-token-file")
+        .text(s"set the access token file, default: ${defaultAccessTokenFile}")
```

Comment:
Do we need something like `See the the docs for how to generate the token file`, or is that obvious?

Reply:
Addressed by https://github.com/digital-asset/daml/pull/3138/commits/ef7a382a50e0bd6dd3e34d72318e6979fb9df764